decentraland/components/presentational/MyAccountProgress.js

```diff
@@ -52,7 +52,7 @@ const styles = StyleSheet.create({
     justifyContent: "flex-start",
   },
   progress: {
-    marginTop: responsiveHeight(10),
+    marginTop: responsiveHeight(1),
   },
```

Comment:
I’m curious if it would look better with 0 margin at the top

Reply:
How its looking like with `marginTop: 0`

![WhatsApp Image 2019-04-16 at 3 05 36 PM](https://user-images.githubusercontent.com/29674218/56216364-33449800-6059-11e9-8d80-063a0190f0b9.jpeg)

Reply:
I think that’s better. Especially if we found a way to hide the gray line at the bottom of the header

Reply:
Oh, the rounding of the bar is showing up on the left for you. I think we should increase the width of the progress bar from 101 to 103 (a slightly inaccurate hack) or find an alternate way to remove the rounding.

Reply:
It sounds like this would work:

```js
navigationOptions = {
  header: {
    style: { shadowColor: 'transparent' },
  },
};
```

https://github.com/react-navigation/react-navigation/issues/865

Reply:
Done.

Reply:
Thanks! Resolving this